rest/data/task.go

```diff
@@ -263,7 +168,7 @@ type TaskFilterOptions struct {
 
 // FindTasksByVersion gets all tasks for a specific version
 // Results can be filtered by task name, variant name and status in addition to being paginated and limited
-func (tc *DBTaskConnector) FindTasksByVersion(versionID string, opts TaskFilterOptions) ([]task.Task, int, error) {
+func FindTasksByVersion(versionID string, opts TaskFilterOptions) ([]task.Task, int, error) {
```

Comment:
This isn't used anymore since we're calling GetTasksByVersion directly so we can just remove it.